apps/client/src/components/Form.tsx

```diff
@@ -0,0 +1,35 @@
+import React from "react";
+
+import { StyledForm } from "./FormStyles";
+
+export type FormProps = {
+  dataId?: string;
```

Comment:
We don't need children?

Reply:
`children` and `className` are defined in `React.HTMLAttributes<HTMLElement>` so we don't need that here